src/styles/utils/_animation.scss

```
@@ -0,0 +1,9 @@
+@keyframes move {
+  from {
+    transform: rotate(0deg);
+  }
+
+  to {
+    transform: rotate(360deg);
+  }
+}

```

You can shorthen the code just by making a 100% rule
```suggestion
@keyframes move {
  100% {
    transform: rotate(360deg);
  }
}

```